### PR TITLE
[MOSIP_32756] adding Readme changes.

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ To deploy Registration Processor services on Kubernetes cluster using Dockers re
 Refer to the [configuration guide](docs/configuration.md).
 
 ## Test
```
Comment (ashok-ksharma) [inline README.md:114]: **Registration processor in sandbox** link is not correct.

File: README.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ To deploy Registration Processor services on Kubernetes cluster using Dockers re
...
-Automated functional tests available in [Functional Tests repo](https://github.com/mosip/mosip-functional-tests)
+Automated functional tests available in [DSL Automation](https://docs.mosip.io/1.2.0/test-automation/dsl-test-rig-automation)
```
Comment (ashok-ksharma) [inline README.md]: The link is wrong. It is about the documentation not about the actual test scripts. Please change the same.

File: README.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ To deploy Registration Processor services on Kubernetes cluster using Dockers re
...
 
 ## APIs
 API documentation is available [here](https://docs.mosip.io/1.2.0/api)
```
Comment (ashok-ksharma) [inline README.md]: API documentation link is not working.

**Review by ashok-ksharma — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Khuddusshariff0022) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (Khuddusshariff0022) [reply README.md]: added this link.  https://[mosip.github.io/documentation/1.2.0/1.2.0.html](https://mosip.github.io/documentation/1.2.0/1.2.0.html)

**Review by ashok-ksharma — APPROVED** [review]

**Pull request merged**